src/api/useTerminologyServiceApi.ts

```diff
@@ -110,6 +120,22 @@ export class TerminologyServiceApi {
       );
     }
   }
+
+  async getAllCodeSystems(): Promise<CodeSystem[]> {
+    try {
+      const response = await axios.get(
+        `${this.baseUrl}/terminology/get-code-systems`,
+        {
+          headers: {
+            Authorization: `Bearer ${this.getAccessToken()}`,
+          },
+        }
+      );
+      return response.data;
+    } catch (err) {
+      console.error("Error retrieving getAllCodeSystems: ", err);
```

Comment:
I think we need error handling here, Looks like if the getAllCodeSystems call fails, we just return an empty list. I wonder if we should let the user known that our call failed. But our story doesn't specify a negative scenario.

Reply:
You can resolve this, I just thought we may need some design decisions or questions to POs regarding the error handling.